extensions/amp-mathml/0.1/test/test-amp-mathml.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.realWin('amp-mathml', {
```

Comment:
for testing, I recommend writing end-2-end integration tests. https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/0.1/test/integration/test-amp-sidebar.js is a good one to use as reference.

In short, the test will essentially render an AMP page with bunch of `amp-mathml` tags you define. Then inside of the test, use various utilities like `poll` to wait for those `math-ml` tags to "render" and "resize". Unfortunately because they are rendered in an iframe, it is not possible to assert that MathML itself rendered but at minimum you can assert that iframe was created and resized to something bigger than 1px.